src/02-common-principles.md

```diff
@@ -153,6 +153,22 @@ A summary of all entities in BIDS and the order in which they MUST be
 specified is available in the [entity table](./99-appendices/04-entity-table.md)
 in the appendix.
 
+### Parametrically linked file collections
```

Comment:
After a bit more thought, I think that complex-valued data should fall under the banner of "file collection", but I'm not sure if magnitude and phase images would be considered "parametrically linked." WDYT? Should the concept be broadened or do complex data count as "parametrically linked"?

Reply:
I think that the key is that separate files are linked in any sense, though, right? The collection idea is that multiple files (from the same data type) either come from the same acquisition or can be treated as if they were from the same acquisition, based on the entities used. In that sense, parametric links and natural links both fall under the umbrella of "file collection."

Reply:
@tsalo I see your point. I can edit the text to include natural links, in that case it'd be useful to reconsidering the title. 

From `Parametrically linked file collections` to: 
* Collections of interdependent files 
* Entity-linked file collections 
* Other suggestions?

Reply:
@tsalo any comments re alternative titles I suggested above? I will edit the text in that regard.

Reply:
Ah, sorry, I don't know how I missed your last response. I like "entity-linked file collections", but "collections of interdependent files" could be a good backup depending on how the rest of the BIDS community feels.

Reply:
Cool, I am editing the text with that title, let's keep other options in our pocket then. Thanks!

Reply:
Okay, I gave it another stab, I guess the inclusivity of the principle now has a better balance.

Reply:
@tsalo if you think that these changes address your concerns, then I'll merge the PR to the `master` of this repo, then make the upstream PR. 

@Gilles86 @KirstieJane @ChristophePhillips @lazaral @tibourauer please add comments if you have suggestions. Thank you!

Reply:
Hi @agahkarakuzu and @tsalo ,

Happy with the `Entity-linked file collections` naming and even better if this makes sens for other modalities. I guess anatomical MRI is not the only modality where some data files are "logically bound" together. 
For example I am thinking of the various field-map files acquired during the MPM protocol to account not only of B0 field inhomogeneity but also the antenna transmit and receive sensitivity (B1+/B1-).

Reply:
> I guess anatomical MRI is not the only modality where some data files are "logically bound" together

This is indeed one of the main motivations to move this concept to the common principles. 

I figure that you are referring to the addition of the [B1+ mapping suffixes](https://github.com/bids-bep001/bep001-v1/blob/master/src/99-appendices/09-qmri.md#grouping-suffixes-for-rf-field-mapping) we introduced in the first pull request to this new appendix of entity linked collections right?